### ISSUE TITLE
Moonlight embebbed 2.2.2 don't work 

### ISSUE THREAD

**Issue by tocco93** [recalbox/recalbox-os#1147]
### What happened
Update pc version of GFE NVidia Geforce Experience version: 3.2.0.96
Moonlight Embedded version: 2.2.2
start streaming but work only the audio with black screen .

### What should have happened
Streaming work perfect 

### Enumerate your devices
Raspberry pi 2 with recalbox 4.0

### How to reproduce the bug
Update your GFE and try.

**Comment by substring** [comment]
indeed, looks like GFE 3.2 changed again something ... it's anyway a constant game of hide and seek with them

**Comment by tocco93** [comment]
for resolve i reinstall GFE 3.1.2.31

**Comment by tocco93** [comment]
New version of moonlight embebbed  is released 
https://github.com/irtimmer/moonlight-embedded/releases

**Comment by substring** [comment]
@tocco93 thank you for the info :) But this won't be updated before 4.1

**Comment by panowan** [comment]
hello i'm a beginner in recalbox, how can i update moonlight 2.2.2 to 2.2.3?
i don't know witch command line to use, in putty...

**Comment by tocco93** [comment]
You can't update moonlight whith command line(a lot of fuction are not included in realbox like sudo, mkdir etc....). You can attempt recalbox 4.1 or downgrade your GFE.

**Comment by panowan** [comment]
Ok thx, i will try to downgrade gfe.

**Comment by IgorCastilheiro** [comment]
@panowan downgrading GFE could be a pain in the ass, my recommendation is to add the line `127.0.0.1 international-gfe.download.nvidia.com` to the hosts file and reboot.
On windows 10 its located under **C:\Windows\System32\drivers\etc.**
It will prevent GFE to update forever.
To bring update functionality back, just remove that line and reboot, GFE should look for update right on reboot.

**Comment by substring** [comment]
Pretty bad news for moonlight 2.2.3 so far : it requires libcec 4, and this breaks KODI

**Comment by an-nix** [comment]
Hi, it's possible to patch moonlight 2.2.3 to rollback to libcec 3. I've tried and it seems to work for me.

https://github.com/an-nix/recalbox-buildroot/tree/master/package/moonlight-embedded

**Comment by substring** [comment]
@an-nix Hey thanks ! I thought about this (or totally disable CEC from the CMakeLists hahaha)
Well you spare me some work ;) Gonna test it !

**Comment by WhitecastlePT** [comment]
Hello i'm having this exact problem, my GFE is the latest by this time version 3.5.0.69. 
I have installed in my RBPi3 Latest recalbox with Moonlight embembed 2.2.2 ...

Do i kneed to upgrade moonlight ? How can i do this ? I'm very new to this....

**Comment by substring** [comment]
Moonlight won't be updated in 4.0.x. The solution has already been posted : https://github.com/recalbox/recalbox-os/issues/1147#issuecomment-267833345

Now this is not the place to ask for support :) Please join the forum and continue the discussion over there.
